App.test.js

```diff
@@ -4,6 +4,6 @@ import App from './app/App';
 import renderer from 'react-test-renderer';
 
 it('renders without crashing', () => {
-  const rendered = renderer.create(<App />).toJSON();
-  expect(rendered).toBeTruthy();
+  // const rendered = renderer.create(<App />).toJSON();
+  expect(true).toBeTruthy();
```

Comment:
I think the test was failing before due to some sort of parsing error with Travis not recognizing ES6 notation. Just wanted to check if this would fix the issue at all but it seems to still be failing, so we need to find a way to configure Travis to parse ES6 notation. I'll change it back to what we had before.